### PR TITLE
Added support for Build Name Updater plugin

### PR DESCRIPTION
**Pull request by erenatas**
I have added support for plugin: buildnameupdater

Also written a test case as well.

### CODE REVIEW

**Comment by ktdreyer** [comment]
This is great. Thanks!

Would you mind rebasing this onto the latest master and squashing it down into one commit?

**Comment by erenatas** [comment]
> This is great. Thanks!
> 
> Would you mind rebasing this onto the latest master and squashing it down into one commit?

Done!

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 /dist
 /output
 /venv
+/.eggs
```
Comment (romanek-adam) [inline .gitignore:10]: please remove this line, it's has already been added above

File: jenkins_job_wrecker/modules/builders.py [hunk 2]
```diff
@@ -102,3 +102,20 @@ def batchfile(child, parent):
...
+        if build_name_element.tag == "buildName":
+            build_name["name"] = build_name_element.text
+        if build_name_element.tag == "macroTemplate":
+            build_name["template"] = build_name_element.text
+        if build_name_element.tag == "fromFile":
+            build_name["file"] = (build_name_element.text == 'true')
+        if build_name_element.tag == "fromMacro":
+            build_name["macro"] = (build_name_element.text == 'true')
+        if build_name_element.tag == "macroFirst":
+            build_name["macro-first"] = (build_name_element.text == 'true')
```
Comment (romanek-adam) [inline jenkins_job_wrecker/modules/builders.py:119]: Please change the above "if"s to "elif"s and add an "else" which would throw NotImplementedError

Comment (erenatas) [reply jenkins_job_wrecker/modules/builders.py:119]: Done

**Comment by ktdreyer** [comment]
I've pushed v1.9.0 to PyPI with this change.

**Pull request merged**